packages/mars-theme/src/components/post.js

```diff
@@ -141,6 +141,49 @@ const Content = styled.div`
     text-decoration: underline;
   }
 
+  // Add input fields styles.
```

Comment:
This comment is not valid so please either remove it or use a CSS comment syntax: `/* Add input fields styles. */`